internal/cli/iam/organizations/users/list.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package users
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const atlasUserListTemplate = `ID	FIRST NAME	LAST NAME	EMAIL{{range .Results}}
```

Comment:
for the sake of having a single template I'm ok with updating the om client to return the full object, ie including the Result field

Reply:
also could we match this to the describe command? we had username there I think, this to avoid surprises from users and offer a consistent view

Reply:
see https://github.com/mongodb/go-client-mongodb-ops-manager/pull/46

Reply:
https://github.com/mongodb/go-client-mongodb-ops-manager/pull/47

Reply:
oh sorry @gssbzn I didn't see you PR 😖  Thanks for that